util/src/network/host.rs

```diff
@@ -837,9 +837,9 @@ impl<Message> Host<Message> where Message: Send + Sync + Clone {
 				let mut s = session.lock().unwrap();
 				if !s.expired() {
 					if s.is_ready() {
+						self.num_sessions.fetch_sub(1, AtomicOrdering::SeqCst);
```

Comment:
This needs to be decremented once per session. Works now cause we have only one handler, but might break future code.